facdb/sql/_create_facdb_spatial.sql

```diff
@@ -41,7 +41,7 @@ with boundary_geosupport as (
 SELECT
 	uid,
 	borocode,
-	zipcode,
+	NULLIF(NULLIF(LEFT(zipcode, 5), ''), '0') as zipcode,
```

Comment:
There is a record from dcla_culturalinstitutions where the zipcode value is 'NY' and this is not getting caught in this logic.  Perhaps change is so that zip code is set to null if it's not a 5 digit number?